src/core/reducers/addons.js

```diff
@@ -119,10 +122,17 @@ export function createInternalAddon(
 
     // These are custom properties not in the API response.
 
-    // TODO: remove this if possible. I think it was added by mistake
-    // but there are some things relying on it :/
+    // TODO: remove this if possible. This is used by core/installAddon
+    // and DiscoPane components for some reason.
     iconUrl: apiAddon.icon_url,
 
+    installURLs: {
```

Comment:
I'm not sure if that will be the best way to do it. It would mean we'd have to pass a version object to all installer components which I'd like to avoid.

Reply:
Ah, fair enough. I'd think maybe `current_version` could be an alias and the default but I don't want things to get too complicated. If you have a cleaner idea in mind: 👍